src/node/coinstats.cpp

```diff
@@ -32,8 +32,7 @@ static void ApplyStats(CCoinsStats &stats, CHashWriter& ss, const uint256& hash,
     ss << VARINT(0u);
 }
 
-//! Calculate statistics about the unspent transaction output set
```

Comment:
Could squash this commit. A propos the commit message, it's unclear what this comment is a duplicate of; unneeded or redundant with the name of the boolean function might be clearer if that is the intended meaning.